pkg/operator/ceph/object/controller_test.go

```diff
@@ -330,4 +330,28 @@ func TestCephObjectStoreController(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, cephv1.ConditionReady, objectStore.Status.Phase, objectStore)
 	logger.Info("PHASE 3 DONE")
+
+	// Test the functionality of verifyObjectUserCleanup
+	// function returns true if there are no cephobjectusers,
+	// otherwise false
+	logger.Info("STARTING PHASE 4")
+	_, userexists := r.verifyObjectUserCleanup(objectStore)
+	assert.True(t, userexists)
+	logger.Infof("Creating user %v", name)
+	objectUser := &cephv1.CephObjectStoreUser{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: cephv1.ObjectStoreUserSpec{
+			Store: store,
+		},
+		TypeMeta: metav1.TypeMeta{
+			Kind: "CephObjectStoreUser",
+		},
+	}
+	objectUser, _ = r.context.RookClientset.CephV1().CephObjectStoreUsers(objectStore.Namespace).Create(objectUser)
+	_, userexists = r.verifyObjectUserCleanup(objectStore)
+	assert.False(t, userexists)
+	logger.Info("PHASE 4 DONE")
```

Comment:
nit: The log messages should indicate what happened, rather than a phase step number.

Reply:
Done